lib/nameifier.go

```diff
@@ -24,23 +22,30 @@ func hash(s string, max int) uint32 {
 	return h.Sum32() % uint32(max)
 }
 
-func loadJson(file []byte, nameifier *Nameifier) {
+func loadJson(file []byte, nameifier *Nameifier) error {
 	err := json.Unmarshal(file, &nameifier)
 	if err != nil {
-		log.Errorf("Eror parsing json from file: %s", err)
+		return err
 	}
+	return nil
 }
 
 type Nameifier struct {
 	Nouns      []string
 	Adjectives []string
 }
 
-func NewNameifier() *Nameifier {
+func NewNameifier() (*Nameifier, error) {
```

Comment:
I'm personally not a fan of returning `error`s from `New()`. I suggest a separate step to load the data into a Nameifier if we could return errors. But since that's the whole function, then we should replace it with something like a function on the Nameifier to load the dictionaries.